### PR TITLE
Implemented AsSpan on NetMQFrame

### PR DESCRIPTION
**Pull request by dxdjgl**

### CODE REVIEW

File: src/NetMQ/NetMQFrame.cs [hunk 1]
```diff
@@ -337,5 +337,14 @@ public byte[] ToByteArray(bool copy = false)
...
             return byteArray;
         }
+
+        /// <summary>
+        /// Return an readonly span of bytes that carries the content of this NetMQFrames Buffer.
+        /// </summary>
+        /// <returns>the Buffer as a readonly span</returns>
+        public ReadOnlySpan<byte> ToReadOnlySpan()
+        {
+            return new ReadOnlySpan<byte>(Buffer);
```
Comment (drewnoakes) [inline src/NetMQ/NetMQFrame.cs:347]: This should take the message length into account. The buffer could be longer than the message.

File: src/NetMQ/NetMQFrame.cs [hunk 2]
```diff
@@ -337,5 +337,14 @@ public byte[] ToByteArray(bool copy = false)
 
             return byteArray;
         }
+
+        /// <summary>
+        /// Return an readonly span of bytes that carries the content of this NetMQFrames Buffer.
+        /// </summary>
+        /// <returns>the Buffer as a readonly span</returns>
+        public ReadOnlySpan<byte> ToReadOnlySpan()
```
Comment (drewnoakes) [inline src/NetMQ/NetMQFrame.cs:345]: What do you think about calling this AsSpan, to match similar methods names in the BCL, such as on strings?

**Review by drewnoakes — COMMENTED** [review]
Great idea, thanks.

**Comment by dxdjgl** [comment]
You are absolutely right, review comments has been fixed.

**Comment by dxdjgl** [comment]
@drewnoakes would you take another look at the code and let me know if other changes are needed.

**Review by drewnoakes — APPROVED** [review]
Apologies for the delay here. Looks great to me.

**Comment by drewnoakes** [comment]
Available in https://github.com/zeromq/netmq/releases/tag/4.0.1.13

**Comment by dxdjgl** [comment]
Thanks Drew for your help

**Pull request merged**